pr47-core/src/vm/al31f/executor/async_executor.rs

```diff
@@ -265,7 +318,37 @@ pub async unsafe fn vm_thread_run_function<A: Alloc>(
                     return Ok(ret_vec);
                 }
             }
-            Insc::FFICallTyck(_, _, _) => {}
+            Insc::FFICallTyck(ffi_func_id, args, ret_value_locs) => {
```

Comment:
Are you sure? I think testing with something like `Raise` could be better.

Reply:
Acknowledged